### DISCUSSION TITLE
Mxnet error

### DISCUSSION THREAD

**Discussion by pratikgehlott** [awslabs/gluonts#1561] [Q&A]
do you guys ever get this error?

MXNetError: Traceback (most recent call last):
  File "../src/operator/./slice_channel-inl.h", line 203
MXNetError: Check failed: dshape[real_axis] % param_.num_outputs == 0U (1 vs. 0) : You are trying to split the 1-th axis of input tensor with shape [32,11] into num_outputs=10 evenly sized chunks, but this is not possible because 10 does not evenly divide 11
@mbohlkeschneider , @lostella ??

**Comment by mbohlkeschneider** [comment]
Hi @Pratik325. No, I have not seen this before. Can you share a minimal reproducible example so we can reproduce the error? 

Thank you!

**Reply by pratikgehlott** [reply]
how should my training data look like 1 or 2?

1) 
![image](https://user-images.githubusercontent.com/65299118/121910734-b7375f00-cd4c-11eb-8295-fc1911b3ba6b.png)


2) 
![image](https://user-images.githubusercontent.com/65299118/121910826-d0d8a680-cd4c-11eb-8f77-449eb404e8c6.png)

**Reply by pratikgehlott** [reply]
hey @mbohlkeschneider ?

**Reply by pratikgehlott** [reply]
I am unable to make reproducible code...



@mbohlkeschneider

**Reply by pratikgehlott** [reply]
can I send you my implementation???

**Reply by mbohlkeschneider** [reply]
> hey @mbohlkeschneider ?

One comment on the expectation on quick responses: Most of us are working in CET time zone and might not work in the evening (currently 6 pm here). We would kindly ask you to consider that when asking for a quick turnaround.

**Reply by mbohlkeschneider** [reply]
> can I send you my implementation???

I would prefer if you would paste it here so everyone would benefit when reading this discussion.

**Reply by pratikgehlott** [reply]
> > hey @mbohlkeschneider ?
> 
> One comment on the expectation on quick responses: Most of us are working in CET time zone and might not work in the evening (currently 6 pm here). We would kindly ask you to consider that when asking for a quick turnaround.

sorry wont happen again.
